### PR TITLE
Update and rename study-sites.md to location-ids.md

### PR DESCRIPTION
**Pull request by goddesswarship**
Removed stray punctuation from Docwriter migration 
Replaced "study sites" with "location ID"
Formatted JSON code
Resized screenshots
Replaced link to locationID.json in repository with a downloadable file.

### CODE REVIEW

File: docs/assets/templates/locationID_template.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
+{
+	"description": "Hierarchical Encounter.locationID list in Wildbook",
+	"source": "http://en.wikipedia.org/wiki/List_of_seas",
```
Comment (TanyaStere42) [inline docs/assets/templates/locationID_template.json]: not gonna stop the PR for this, but we gotta remove "source" or at least make it accurate. This is not a List of seas from wikipedia 😆

File: docs/data/location-ids.md [hunk 2]
```diff
@@ -16,22 +16,46 @@ When you [report an Encounter](report-encounter.md), you can set its **location
 
 ![match Criteria](../assets/images/wb-encounter-customalg.png)
 
-## Adding a New Study Site
+## Adding a Location ID
 
-To add a new study site, you'll need to contact your Wildbook's administrator or make the request at the [Wildbook Community](https://community.wildme.org).
+To add a new Location ID, you'll need to contact your Wildbook's administrator or make the request at the [Wildbook Community](https://community.wildme.org).
```
Comment (TanyaStere42) [inline docs/data/location-ids.md]: should we add that they can make a PR against their platform branch? or handle that in a different page?

**Review by TanyaStere42 — APPROVED** [review]
Comments aren't blocking; just points to think on. I'm gonna approve now and you can either revise or merge immediately

File: docs/data/location-ids.md (hunk 2 above)
Comment (goddesswarship) [reply docs/data/location-ids.md]: Good call! I've added a link to our PR page as a third option.

File: docs/assets/templates/locationID_template.json (hunk 1 above)
Comment (goddesswarship) [reply docs/assets/templates/locationID_template.json]: Fixed!

**Pull request merged**